addOns/sstiscanner/src/main/resources/org/zaproxy/zap/extension/sstiscanner/resources/Messages.properties

```diff
@@ -0,0 +1,17 @@
+# This file defines the default (English) variants of all of the internationalised messages
+
+sstiscanner.name = Zap-ESUP scanner for SSTI
+sstiscanner.desc = This is a scanner for SSTI  in the form of reflected, stored and blind.
+
+sstiscanner.sstiblindplugin.name = Server Side Template Injection (Blind)
+sstiscanner.sstiblindplugin.desc = When the user input is inserted in the template instead of being used as argument in rendering is evaluated by the template engine. Depending on the template engine it can lead to remote code execution.
+sstiscanner.sstiblindplugin.soln = Instead of inserting the user input in the template, use it as rendering argument
+sstiscanner.sstiblindplugin.refs = https://portswigger.net/blog/server-side-template-injection
+sstiscanner.sstiblindplugin.alert.recvdcallback.attack = Received callback from the server
+sstiscanner.sstiblindplugin.alert.timedelay.attack = Value [{0}] caused a time delay in field: [{1}] at [{2}]  
```

Comment:
The first one should have a period.

Both should be used in the otherinfo field instead of attack.